libraries/botbuilder-dialogs-adaptive-teams/tests/tests/actionTests.test.ts

```diff
@@ -0,0 +1,469 @@
+// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+import {
+    ComponentRegistration,
+    ConversationState,
+    TestAdapter,
+    useBotState,
+    MemoryStorage,
+    UserState,
+    Channels,
+    ConversationReference,
+    ChannelAccount,
+    ConversationAccount,
+} from 'botbuilder';
+import { ResourceExplorer } from 'botbuilder-dialogs-declarative';
+import { TeamsComponentRegistration } from '../../lib';
+import { AdaptiveTestComponentRegistration, TestUtils } from 'botbuilder-dialogs-adaptive-testing';
+import { AdaptiveComponentRegistration } from 'botbuilder-dialogs-adaptive';
+import { ConnectorClient, MicrosoftAppCredentials } from 'botframework-connector';
+import { ok } from 'assert';
+import path = require('path');
+import nock = require('nock');
+import { jwt } from 'botbuilder-test-utils';
+
+jwt.mocha();
```

Comment:
This needs to go inside a `describe` block otherwise (AFAIK) it gets added to all mocha tests.

Reply:
yeah, that's probably correct. Side-note: should we export it as something outside of jwt, as it's own method? Seems odd to have non-JWT-related tests import something starting with JWT.

Reply:
Ah - I didn't review closely enough. There should probably be a `nock` specific helper that does all the mocha registrations. `jwt`, itself, could then use that. I'll take care of that at some point. Good catch!

Reply:
Note: I did make the change and `jwt.mocha()` falls under `describe()`